apps/web/src/views/PositionManagers/hooks/useFilters.ts

```diff
@@ -7,18 +7,26 @@ function filterHookFactory(filterName: string) {
     const router = useRouter()
     const filter = useMemo(() => {
       const value = router.query[filterName]
-      if (Array.isArray(value)) {
-        return value[0]
+      if (value) {
+        if (Array.isArray(value)) {
+          return decodeURIComponent(value[0])
+        }
+        return decodeURIComponent(value)
       }
-      return value
+      return undefined
     }, [router.query])
 
     const setFilter = useCallback(
       (value?: string) => {
         router.push(
-          updateQuery(router.asPath, {
-            [filterName]: value,
-          }),
+          updateQuery(
+            router.asPath,
+            value
+              ? {
+                  [filterName]: encodeURIComponent(value.trim()),
```

Comment:
`updateQuery` will do the encode so no need to encode again

Reply:
<img width="1788" alt="image" src="https://github.com/pancakeswap/pancake-frontend/assets/116779127/f9afa506-5cae-485b-97cb-b907d5241332">

Just tried on prod seems ok?